src/detect-engine-alert.h

```diff
@@ -28,8 +28,6 @@
 #include "decode.h"
 #include "detect.h"
 
-void AlertQueueInit(DetectEngineThreadCtx *det_ctx);
-void AlertQueueFree(DetectEngineThreadCtx *det_ctx);
```

Comment:
Since these are defined in `src/detect-engine-alert.c`, I think it's better to keep these declarations, remove the duplicate ones in the other file, and then add this hear as an include.